### PR TITLE
Updated triggers (#2719)

### LINKED ISSUE #2719: Rust Cargo Packages: should accept the term "crate"
The unit of compilation in Rust is a crate, and packages can only contain a single library crate, so when searching, people are frequently going to want to search for a "rust crate" or "cargo crate". You should be able to use "cargo crate time" in the same way you would use "cargo package time".

------
IA Page:  http://duck.co/ia/view/rust_cargo 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @TomBebbington

### PR DESCRIPTION
**Pull request by pjhampton**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`Rust Cargo Packages: Adds two new triggers`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{SpiceRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Adds the triggers **rust crate** or **cargo crate**

###### Which issues (if any) does this fix?

Fixes #2719

###### People to notify (@mention interested parties)

@GuiltyDolphin, @lambda, @tagawa, @TomBebbington

---

https://duck.co/ia/view/rust_cargo

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @TomBebbington

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pjhampton Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Spice/RustCargo.pm [hunk 1]
```diff
@@ -4,7 +4,7 @@ package DDG::Spice::RustCargo;
 use strict;
 use DDG::Spice;
 
-triggers startend => 'cargo package', 'cargo packages', 'rust package', 'rust packages', 'rust cargo';
+triggers startend => 'cargo package', 'cargo packages', 'rust package', 'rust packages', 'rust cargo', 'rust crate', 'cargo crate';
```
Comment (GuiltyDolphin) [inline lib/DDG/Spice/RustCargo.pm]: Lets format these a bit so it doesn't go off the screen.

**Comment by TomBebbington** [comment]
Oh hey I didn't even know my changes got merged even though I forgot... that's nice to see.

**Comment by tagawa** [comment]
Thanks for the quick PR. I'm not sure about "cargo crate" though. Couldn't that lead to over-triggering?

**Comment by pjhampton** [comment]
In all honesty, I'm not really all that familiar with rust or their community, but if it makes sense to them I don't see why not. In terms of over-triggering, it's hard to give an objective answer. Although somewhat ambiguous, I would say "cargo crate" is an unlikely bigram and shouldn't be worried about. But your say is final @tagawa 😄

**Comment by tagawa** [comment]
Hmm, looks like "cargo crate" is used sometimes included for a product called Firefly Cargo Crate, so I think we should play it safe. Please could the "cargo crate" be remove?

**Comment by pjhampton** [comment]
Yeah, sure thing. I'll update this pr soon :)

**Comment by tagawa** [comment]
@pjhampton A thousand thanks! Merged.

**Pull request merged**